src/pymor/algorithms/image.py

```diff
@@ -212,7 +212,7 @@ def estimate_image_hierarchical(operators=(), vectors=(), domain=None, extends=N
         image.append(new_image, remove_from_other=True)
         if orthonormalize:
             with logger.block('Orthonormalizing ...'):
-                gram_schmidt(image, offset=gram_schmidt_offset, product=product, copy=False)
+                rrqr(image, offset=gram_schmidt_offset, product=product, copy=False)
```

Comment:
Something is appended to `image` in line 212, so maybe it should be as is.

Reply:
Oh I see, I just thought it would be good to remove the specific reference to `gram_schmidt`since now it could be also a different algorithm. Maybe just `offset` would do?

Reply:
Aha, yes, that's a good point. I changed it now.